docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -89,27 +91,27 @@ we need to create an `.env` file in the same place as the ipynb notebook.
 
 Save your Twitter API Bearer token in the `.env` file:
 
-```
+```bash linenums="1" hl_lines="4-4" title="someting.env"
```

Comment:
same here: python

Reply:
this should be bash since it's a .env file?